src/Witness/NTS.Witness.Web/Properties/launchSettings.json

```diff
@@ -0,0 +1,39 @@
+{
+  "profiles": {
```

Comment:
Use the same launch profiles pattern we use for Judge, Nexus - Localhost, Staging etc